packages/js/images/yoast_loading_spinner.svg

```diff
@@ -0,0 +1 @@
+<svg id="Yoast_Spinner" data-name="Yoast Spinner" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 229.95 98"><defs><style>.cls-1{fill:#e7c9da;}.cls-2{fill:#a4296a;}</style><symbol id="Spinner" data-name="Spinner" viewBox="0 0 98 98"><path class="cls-1" d="M49,0A49,49,0,1,0,98,49,49,49,0,0,0,49,0Zm0,89A40,40,0,1,1,89,49,40,40,0,0,1,49,89Z"/><path class="cls-2" d="M91.44,73.5A49,49,0,0,1,49,98V89A40,40,0,0,0,83.65,69Z"/></symbol></defs><use width="98" height="98" xlink:href="#Spinner"/><path class="cls-1" d="M221.45,137.3a24,24,0,1,0,24,24,24,24,0,0,0-24-24Zm0,43.59A19.59,19.59,0,1,1,241,161.3a19.59,19.59,0,0,1-19.59,19.59Z" transform="translate(-65.23 -112.3)"/><path class="cls-2" d="M242.23,173.3a24,24,0,0,1-20.78,12v-4.41a19.6,19.6,0,0,0,17-9.79Z" transform="translate(-65.23 -112.3)"/><path class="cls-1" d="M283.18,149.3a12,12,0,0,0-.11,24h.11a12,12,0,0,0,.1-24Zm0,21.8h-.09a9.8,9.8,0,0,1,.09-19.6h.08a9.8,9.8,0,0,1-.08,19.6Z" transform="translate(-65.23 -112.3)"/><path class="cls-2" d="M289.27,151a12,12,0,0,1,5.91,10.45l-2.21,0a9.76,9.76,0,0,0-4.82-8.53Z" transform="translate(-65.23 -112.3)"/></svg>
```

Comment:
Not sure if this should have an ID even (or `data-name` for that matter). But the capitalization and underscore seems wrong to me 😅 
And missing the normal `role="img" aria-hidden="true" focusable="false"` attributes that the other SVGs have.